megamek/src/megamek/client/ui/swing/tooltip/UnitToolTip.java

```diff
@@ -970,12 +985,43 @@ public static String getSensorDesc(Entity e) {
                 + srh.maxSensorRange + ")";
     }
 
+    private static String getDamageLevelDesc(Entity entity) {
+        String result;
+        switch (entity.getDamageLevel()) {
+            case Entity.DMG_CRIPPLED:
+                String msg_crippled = ' '+Messages.getString("BoardView1.Tooltip.Crippled");
+                result = guiScaledFontHTML(GUIP.getWarningColor()) + msg_crippled + "</FONT>";
+                break;
+            case Entity.DMG_HEAVY:
+                String msg_heavydmg = Messages.getString("BoardView1.Tooltip.HeavyDmg");
+                result = ' '+guiScaledFontHTML(GUIP.getWarningColor()) + msg_heavydmg + "</FONT>";
+                break;
+            case Entity.DMG_MODERATE:
+                String msg_moderatedmg = Messages.getString("BoardView1.Tooltip.ModerateDmg");
+                result = ' '+msg_moderatedmg;
+                break;
+            case Entity.DMG_LIGHT:
+                String msg_lightdmg = Messages.getString("BoardView1.Tooltip.LightDmg");
+                result = ' '+msg_lightdmg ;
+                break;
+            default:
+                String msg_undamaged = Messages.getString("BoardView1.Tooltip.Undamaged");
+                result = "&nbsp;&nbsp;" + msg_undamaged;
+                break;
+        }
+        return result;
+    }
+
     /** Returns values that only are relevant when in-game such as heat. */
-    private static StringBuilder inGameValues(Entity entity, Player localPlayer) {
+    private static StringBuilder inGameValues(Entity entity, Player localPlayer, boolean inGameValue) {
         Game game = entity.getGame();
         boolean isGunEmplacement = entity instanceof GunEmplacement;
         String result = "";
 
+        if (!inGameValue) {
+            return new StringBuilder().append("");

```

Comment:
Why bother appending empty string? Doesn't it start initialized as an empty string?